doc/Accessibility.md

```diff
@@ -0,0 +1,124 @@
+# Accessibility
+
+## SimpleAccessibility
+
+The App template makes use of Simple Accessibility mode which resolves some issues with focus nesting and also makes ScreenReaders work a lot more similarly on different platforms. further documentation can be found [here](https://platform.uno/docs/articles/features/working-with-accessibility.html#simpleaccessibility-mode) 
+
+## Supported features
+
+The way screenreaders will behave within the app is relatively simple, When first accessing a new page the focus will generally be "grabbed" by the topmost element of the page. After which the user can cycle through elements by swiping left/right on their device. The screenreader will then focus the next eligible element.
+
+For an element to be focusable, it needs to have an accessibilityTrait, Currently in UNO, as listed [here](https://platform.uno/docs/articles/features/working-with-accessibility.html#known-issues) only the following elements support accessibility by default:
```

Comment:
For an element to be focusable, it needs to have an accessibilityTrait**.** Currently in U**no**, as listed [here](https://platform.uno/docs/articles/features/working-with-accessibility.html#known-issues)**,* only the following elements support accessibility by default: